src/lists.cpp

```diff
@@ -216,6 +217,27 @@ namespace
    };
 }
 
+std::unordered_map<std::string, std::string> paper_titles = [] {
+   std::unordered_map<std::string, std::string> titles;
+   std::ifstream in{"meta-data/paper_titles.txt"};
+   std::string paper_number, title;
+   while (in >> paper_number && std::getline(in, title))
+      titles[paper_number] = title;
+   return titles;
+}();
+
+// The title of the specified paper, formatted as an HTML title="..." attribute.
+std::string paper_title_attr(std::string paper_number) {
+   auto title = paper_titles[paper_number];
+   if (!title.empty())
+   {
+      for (auto p = title.find('"'); p != title.npos; p = title.find('"', p+1))
+         title.replace(p, 1, "&quot;");
```

Comment:
We should probably replace ampersands first. You wouldn't believe the proposal titles that Bobby Tables submits.

Reply:
Done